src/com/gfredericks/test/chuck/cljs_test.cljc

```diff
@@ -0,0 +1,54 @@
+(ns com.gfredericks.test.chuck.cljs-test
```

Comment:
We have separate namespaces exactly because of the separation in test.check: as they have separate ns, we need to refer to `cljs-test.check.properties/for-all` and `clojure-test.check.properties/for-all` in our macro depending if we are compiling for clj or cljs, and we couldn't find the way to do that from a single macro.

Another solution is to have the two macros in the same namespace with different names. That's worst than the current scenario IMO, because it forces the user to use reader conditionals every time he was to use the macro, instead of only in the ns declaration.

So I guess we have two options:
1. Wait for the release of test.check where cljs-test ns is unified with clojure-test.
2. Keep the separate ns. When test.check changes its namespaces, test.chuck will follow (perhaps keeping the cljs-test ns with a deprecation notice until the next major release...)